src/Innovator.Client/Command.cs

```diff
@@ -326,7 +326,7 @@ protected virtual void AddAml(string query)
     /// Create a command from an interpolated string
     /// </summary>
     /// <param name="formatted">Interpolated string to convert to a command</param>
-    public Command(FormattableString formatted)
+    public Command(FormattableString formatted) : this()
     {
       this.WithAml(formatted.Format, formatted.GetArguments());
       _sub.Style = ParameterStyle.CSharp;
```

Comment:
Filter issues and pull requests that involve, but don't necessarily @mention, certain people: state:open type:issue involves:octocat